installation-usage.md

```diff
@@ -81,6 +81,32 @@ spec:
       image: nginx
 ```
 
+#### Base syscalls for a container runtime
+
+An example of the minimum required syscalls for a runtime such as runc to launch
```

Comment:
Let's include the runc version:
```suggestion
An example of the minimum required syscalls for a runtime such as runc v1.0.0-rc92 to launch
```

Reply:
I didn't happen to test it on `v1.0.0-rc92`, have you? I've only tried `v1.0.0-rc10`

Reply:
No, I just assumed the latest version. I would not expect changes in the setup/syscalls, but who knows…

Reply:
tested on rc92, added the version here and to the yaml